components/SpotifyInterface/CMakeLists.txt

```diff
@@ -1,11 +1,7 @@
 idf_component_register(SRCS "SpotifyAPICall.c" "JsonExtraction.c" "SpotifyInterface.c" 
                             "SpotifyHttpLocalServer.c"
-                    INCLUDE_DIRS "include" "lib" "certs"
+                    INCLUDE_DIRS "include" "lib"
                     
-                    EMBED_TXTFILES certs/server_root_cert.pem 
-                                   certs/local_server_cert.pem 
-                                   certs/local_server_key.pem
-
```

Comment:
why you remove this part ?

Reply:
They are not needed anymore, the code uses esp_certificate_bundle.h which gets certificates inside the code